File: src/targets/javascript/jquery/fixtures/multipart-form-data-no-params.js
const settings = {
  async: true,
  crossDomain: true,
<<<<<<< HEAD
  url: 'https://httpbin.org/anything',
=======
  url: 'http://mockbin.com/har',
>>>>>>> upstream/master
  method: 'POST',
  headers: {
    'Content-Type': 'multipart/form-data'
  }
};

$.ajax(settings).done(function (response) {
  console.log(response);
});

File: src/targets/r/httr/fixtures/multipart-form-data-no-params.r
library(httr)

<<<<<<< HEAD
url <- "https://httpbin.org/anything"
=======
url <- "http://mockbin.com/har"
>>>>>>> upstream/master

payload <- ""

response <- VERB("POST", url, body = payload, content_type("multipart/form-data"))

content(response, "text")